menu.html

```diff
@@ -106,7 +106,7 @@ <h1>Kyoto Dining</h1>
                                 sashimi (6 slices)
                             </td>
                             <td>
-                                Euro 450
+                                Euro 650
```

Comment:
I think that is too expensive for sashimi!

Reply:
No, it is actually a good price

Reply:
No one might buy them anymore.